### DISCUSSION TITLE
How To Improve Binder Design in Silico Success Rate?

### DISCUSSION THREAD

**Discussion by xinran1129305** [RosettaCommons/RFdiffusion#177] [Q&A]
Hi there.
I got around 150 binder designs(backbones)of a specific target protein, but none of the designs meet the requirement of pAE<10. I tried different hotspot and length of binder, but they didn't work out and just remained pAE~26(some of them could reach 19). In this way, what can I do to improve the success rate of designing?
Thanks in advance!!

**Comment by joaojuniorCTC** [comment]
Same problem here...
